extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -197,6 +201,15 @@ export class DataTierApplicationWizard {
 				page.dacFxPage.setupNavigationValidator();
 				page.dacFxPage.onPageEnter();
 			}
+
+			// Telemetry event for each page change, if idxLast page is -1, means the wizard instance just started
+			if (idxLast !== -1) {
+				TelemetryReporter.createActionEvent(TelemetryViews.DataTierApplicationWizard, 'DacpacPageChangeEvents')
```

Comment:
This kind of event would really be beneficial to have in core - it's useful enough for all extensions to have access to this kind of data for their wizards.

Reply:
I have moved this to the core and the navigation between the pages will be sent to telemetry with from and to pages and the wizard name (These wizard name and page names are for telemetry use only and provided as an optional parameters to the wizard and page).